homeassistant/components/netgear/sensor.py

```diff
@@ -52,8 +129,19 @@ async def async_setup_entry(
     """Set up device tracker for Netgear component."""
     router = hass.data[DOMAIN][entry.entry_id][KEY_ROUTER]
     coordinator = hass.data[DOMAIN][entry.entry_id][KEY_COORDINATOR]
-    tracked = set()
 
+    # Router entities
+    router_entities = []
+
+    for attribute in SENSOR_TRAFFIC_TYPES:
```

Comment:
We can just iterate the descriptions and pass the description directly to the entity.

Reply:
done